common-util/calculate7DayAverage.js

```diff
@@ -0,0 +1,8 @@
+export const calculate7DayAverage = (performances) => {
```

Comment:
can you please reuse it in `pages/api/main-metrics.js` instead of get7DaysAverage? I also think `get7DaysPerformanceArray` can be removed from there since we divide by 7 (so missing days will be count as zero anyway), and then we can pass a second param called "key" to understand by which key get the average (activeMultisigCount or count)